runtime-manager/modules/ROOT/pages/deployment-strategies.adoc

```diff
@@ -5,17 +5,21 @@ endif::[]
 :page-aliases: general:getting-started:deploy-to-server.adoc, deploying-to-pcf.adoc
 
 
-image:logo-cloud-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-cloud2-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub 2.0"]
```

Comment:
Should the #target go to a different place for CH2 and CH1? 
Like `#cloudhub2-deployments`

Reply:
For all these icon links, I'm not seeing the destination section. In latest, there are sections that those link to, but I'm not seeing them in this version.

Reply:
These also targeted each of the sections that we removed and replaced with tables. I'll see if there’s somewhere else they can target, or remove them.